src/mission_companion.cpp

```diff
@@ -419,7 +420,7 @@ bool talk_function::display_and_choose_opts( mission_data &mission_key, npc &p,
         if( redraw ) {
             werase( w_list );
             draw_border( w_list );
-            mvwprintz( w_list, 1, 1, c_white, name_mission_tabs( p, id, title, tab_mode ) );
+            mvwprintz( w_list, 1, 1, c_white, talk_function::name_mission_tabs( omt_pos, id, title, tab_mode ) );
```

Comment:
display_and_choose_opts() is in talk_function, so why are specifying the namespace for name_mission_tabs?